lib/registries/addon/components/license-viewer/template.hbs

```diff
@@ -1,30 +1,32 @@
 <OsfDialog as |dialog|>
-    <dialog.trigger>
-        <OsfButton
+    <div local-class='LicenseView'>
```

Comment:
This is addressed in a separate PR too

Reply:
Same here, regarding the changes for ENG-1644.